orchestration/src/main/java/com/sap/ai/sdk/orchestration/OrchestrationResponse.java

```diff
@@ -0,0 +1,35 @@
+package com.sap.ai.sdk.orchestration;
+
+import static lombok.AccessLevel.PACKAGE;
+
+import com.sap.ai.sdk.orchestration.client.model.CompletionPostResponse;
+import com.sap.ai.sdk.orchestration.client.model.LLMModuleResultSynchronous;
+import javax.annotation.Nonnull;
+import lombok.RequiredArgsConstructor;
+import lombok.Value;
+
+/** Orchestration chat completion output. */
+@Value
+@RequiredArgsConstructor(access = PACKAGE)
+public class OrchestrationResponse {
+  CompletionPostResponse data;
+
+  /**
+   * Get the message content from the output.
+   *
+   * <p>Note: If there are multiple choices only the first one is returned
+   *
+   * @return the message content or empty string.
+   * @throws OrchestrationClientException if the content filter filtered the output.
```

Comment:
(Major/Comment)

In case I'm not missing something, and we're ignoring the risk, let's at least document that(?)

```suggestion
   * @throws OrchestrationClientException if the content filter filtered the output.
   * @throws NullPointerException if the payload is incomplete.
```

Reply:
See comment above.